src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/ServerConfigurationFactory.cs

```diff
@@ -34,8 +34,8 @@ public void InitializeConfiguration(ServerConfiguration serverConfiguration)
 
         // Update any other global options based on the configuration the server was started with.
 
-        // Use the SharedDependenciesPath option as a proxy for whether or not devkit is running.
-        var isDevkitEnabled = !string.IsNullOrEmpty(serverConfiguration.SharedDependenciesPath);
+        // Check if the devkit extension is included to see if devkit is enabled.
+        var isDevkitEnabled = serverConfiguration.ExtensionAssemblyPaths.Contains("Microsoft.VisualStudio.LanguageServices.DevKit.dll");
```

Comment:
>Microsoft [](http://example.com/codeflow?start=83&length=9)

This assumes that just the file name is in the paths. `ExtensionAssemblyPaths` should contain absolute paths though, right?

Reply:
oops, fixed